tdlibjson_wrapper/tdlibQt/models/MessagingModel.hpp

```diff
@@ -70,6 +70,8 @@ class MessagingModel : public QAbstractListModel
         FILE_CAPTION,
         PHOTO_ASPECT,
         DOCUMENT_NAME,
+	VOICE_DURATION,
```

Comment:
```suggestion
        VOICE_DURATION
```
Maybe we should just indent this line properly. :b

Reply:
Thanks for the Hint I added it including the comma at the end =)

Reply:
and never ever use tabs :D